### DISCUSSION TITLE
need to get code from github URL to work on it.

### DISCUSSION THREAD

**Discussion by acmedeepak** [community/community#21760] [New to GitHub]
<p>I have recently started working on Angular, looked into a project to get hands on. In Github, There is a project which I want to download and use the code and modify it but I am not sure how?</p>
<p>URL: <a href="https://github.com/mosh-hamedani/organic-shop" rel="nofollow noopener">https://github.com/mosh-hamedani/organic-shop</a></p>
<p>      I downloaded the zip using “Clone/Download”&gt;Download zip button. I got an e2e type of file. Please let me know how I can open the code. Is this something to do with firebase.</p>
<p>      I need to download angular code from there but I found the project file structure and everything to be different. Please look into the link and suggest to me what I need to do to find out the source code.</p>
<p>I need a way to proceed. Help me with this  </p>

**Comment by Wabri** [comment, marked as answer]
<p>Hi <a class="mention" href="/acmedeepak">@acmedeepak</a>,</p>
<p>Welcome to the Github Community Forum!</p>
<p>The source directory is called src and is inside the project, when you unzip the archive you have extract it. Maybe you loose the directory because there are a lot of files, try to create a special folder for that project and unzip there.</p>
<p>-Gabriele-</p>
<p>Mark helpfull posts with Accept as Solution to help other users locate important info. Don’t forget to give Kudos for great contents!</p>
